walkthroughs/week-3-server/examples/server-stats/subtraction-game/src/main/java/com/google/sps/servlets/SubtractionServlet.java

```diff
@@ -0,0 +1,81 @@
+// Copyright 2019 Google LLC
+//
```

Comment:
Ah, I think those files are extra?

This looks like it's from

walkthroughs/week-3-server/examples/*server-stats*/subtraction-game/src/main/java/com/google/sps/servlets/SubtractionServlet.java

you want week-3-step-3/walkthroughs/week-3-server/examples/subtraction-game (no server-stats).

That file does look like it's in the directory, so I think you don't need to add anything! https://github.com/SeanT18/step/tree/week-3-step-3/walkthroughs/week-3-server/examples/subtraction-game

so I think you can git revert the commit that adds those